### PR TITLE
Update 2019 blog content files to move author details in front-matter

### PR DESCRIPTION
**Pull request by dipesh-rawat**
Split out from PR https://github.com/kubernetes/website/pull/45957

This PR extends the changes made in PR https://github.com/kubernetes/website/pull/45865 and implements those changes to all the 2019 blog files as per the recommendation in https://github.com/kubernetes/website/pull/45865#issuecomment-2054036103. The primary change involves relocating author details into the metadata within the front matter.

Useful Link
[Preview Blog Summary Page](https://deploy-preview-46234--kubernetes-io-main-staging.netlify.app/blog)

### CODE REVIEW

File: content/en/blog/_posts/2019-01-14-apiserver-dry-run-and-kubectl-diff.md [hunk 1]
```diff
@@ -2,10 +2,10 @@
 layout: blog
 title: 'APIServer dry-run and kubectl diff'
 date: 2019-01-14
+author: >
+  Antoine Pelisse (Google Cloud, @apelisse)
```
Comment (sftim) [inline content/en/blog/_posts/2019-01-14-apiserver-dry-run-and-kubectl-diff.md:6]: I'd change this to:
```suggestion
  Antoine Pelisse (Google Cloud)
```

File: content/en/blog/_posts/2019-01-15-container-storage-interface-ga.md [hunk 2]
```diff
@@ -2,13 +2,13 @@
 title: Container Storage Interface (CSI) for Kubernetes GA
 date: 2019-01-15
 slug: container-storage-interface-ga
+author: >
+  Saad Ali (Senior Software Engineer, Google)
```
Comment (sftim) [inline content/en/blog/_posts/2019-01-15-container-storage-interface-ga.md:6]: Let's aim to be consistent with how we do other attributions. I recommend this even if that means dropping metadata.
```suggestion
  Saad Ali (Google)
```

File: content/en/blog/_posts/2019-02-12-building-a-kubernetes-edge-control-plane-for-envoy-v2.md [hunk 3]
```diff
@@ -2,15 +2,13 @@
 title: Building a Kubernetes Edge (Ingress) Control Plane for Envoy v2
 date: 2019-02-12
 slug: building-a-kubernetes-edge-control-plane-for-envoy-v2
+author: >
+  Daniel Bryant (Product Architect, Datawire),
+  Flynn (Ambassador Lead Developer, Datawire),
+  Richard Li (CEO and Co-founder, Datawire)
```
Comment (sftim) [inline content/en/blog/_posts/2019-02-12-building-a-kubernetes-edge-control-plane-for-envoy-v2.md:8]:
```suggestion
  Daniel Bryant (Datawire),
  Flynn (Datawire),
  Richard Li (Datawire)
```

File: content/en/blog/_posts/2019-02-28-automate-operations-on-your-cluster-with-operatorhub.md [hunk 4]
```diff
@@ -1,11 +1,10 @@
 ---
 title: Automate Operations on your Cluster with OperatorHub.io
 date: 2019-02-28
+author: >
+  Diane Mueller (Director of Community Development, Cloud Platforms, Red Hat)
```
Comment (sftim) [inline content/en/blog/_posts/2019-02-28-automate-operations-on-your-cluster-with-operatorhub.md:5]:
```suggestion
  Diane Mueller (Red Hat)
```

File: content/en/blog/_posts/2019-03-25-1-14-release-announcement.md [hunk 5]
```diff
@@ -3,10 +3,10 @@ title: 'Kubernetes 1.14: Production-level support for Windows Nodes, Kubectl Upd
 date: 2019-03-25
 slug: kubernetes-1-14-release-announcement
 evergreen: true
+author: >
+  1.14 [Release Team](https://bit.ly/k8s114-team)
```
Comment (sftim) [inline content/en/blog/_posts/2019-03-25-1-14-release-announcement.md:7]:
```suggestion
  [Kubernetes v1.14 Release Team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.14/release_team.md)
```

File: content/en/blog/_posts/2019-03-28-running-kubernetes-locally-on-linux-with-minikube.md [hunk 6]
```diff
@@ -1,8 +1,9 @@
 ---                                           
 title: 'Running Kubernetes locally on Linux with Minikube - now with Kubernetes 1.14 support'                                                           
-date: 2019-03-28                                  
+date: 2019-03-28
+author: >
+  [Ihor Dvoretskyi](https://twitter.com/idvoretskyi) (Developer Advocate, Cloud Native Computing Foundation)
```
Comment (sftim) [inline content/en/blog/_posts/2019-03-28-running-kubernetes-locally-on-linux-with-minikube.md:5]:
```suggestion
  [Ihor Dvoretskyi](https://twitter.com/idvoretskyi) (Cloud Native Computing Foundation)
```

**Review by sftim — COMMENTED** [review]
I recommend dropping the job titles to match how we attribute brand new articles. See some of the suggestions I've made.

File: content/en/blog/_posts/2019-11-05-grokkin-the-docs.md [hunk 7]
```diff
@@ -3,10 +3,10 @@ layout: blog
 title: "Grokkin' the Docs"
 date: 2019-11-05
 slug: Grokkin-the-Docs
+author: >
+  [Aimee Ukasick](https://www.linkedin.com/in/aimee-ukasick/) (Independent Contributor)
```
Comment (sftim) [inline content/en/blog/_posts/2019-11-05-grokkin-the-docs.md:7]: (nit)
```suggestion
  [Aimee Ukasick](https://www.linkedin.com/in/aimee-ukasick/) (independent contributor)
```

File: content/en/blog/_posts/2019-10-29-2019-sig-docs-survey.md [hunk 8]
```diff
@@ -3,10 +3,11 @@ layout: blog
 title: "Kubernetes Documentation Survey"
 date: 2019-10-29
 slug: kubernetes-documentation-end-user-survey
+author: >
+  [Aimee Ukasick](https://www.linkedin.com/in/aimee-ukasick/),
+  SIG Docs
```
Comment (sftim) [inline content/en/blog/_posts/2019-10-29-2019-sig-docs-survey.md:8]:
```suggestion
  and Kubernetes SIG Docs
```

File: content/en/blog/_posts/2019-08-06-OPA-Gatekeeper-Policy-and-Governance-for-Kubernetes.md [hunk 9]
```diff
@@ -4,10 +4,12 @@ layout: blog
 title: "OPA Gatekeeper: Policy and Governance for Kubernetes"
 date: 2019-08-06
 slug: OPA-Gatekeeper-Policy-and-Governance-for-Kubernetes 
+author: >
+  Rita Zhang (Microsoft), Max Smythe (Google),
+  Craig Hooper (Commonwealth Bank AU), Tim Hinrichs (Styra),
+  Lachie Evenson (Microsoft), Torin Sandall (Styra)
```
Comment (sftim) [inline content/en/blog/_posts/2019-08-06-OPA-Gatekeeper-Policy-and-Governance-for-Kubernetes.md:10]: I recommend one author per line.

File: content/en/blog/_posts/2019-06-24-kubeadm-ha-v115.md [hunk 10]
```diff
@@ -2,13 +2,11 @@
 layout: blog
 title: 'Automated High Availability in kubeadm v1.15: Batteries Included But Swappable'
 date: 2019-06-24
+author: >
+  Lucas Käldström ([@luxas](https://github.com/luxas) SIG Cluster Lifecycle co-chair & kubeadm subproject owner, Weaveworks),
+  Fabrizio Pandini ([@fabriziopandini](https://github.com/fabriziopandini) kubeadm subproject owner, Independent)
```
Comment (sftim) [inline content/en/blog/_posts/2019-06-24-kubeadm-ha-v115.md:7]: Always write “independent” in lower case, unless they represent a company named something like **Independent, Inc.**

Comment (sftim) [reply content/en/blog/_posts/2019-06-24-kubeadm-ha-v115.md:7]:
```suggestion
  Lucas Käldström ([@luxas](https://github.com/luxas) (Weaveworks),
  Fabrizio Pandini ([@fabriziopandini](https://github.com/fabriziopandini) (independent)
```

File: content/en/blog/_posts/2019-06-24-kubeadm-ha-v115.md [hunk 11]
```diff
@@ -2,13 +2,11 @@
...
 ---
```
Comment (sftim) [inline content/en/blog/_posts/2019-06-24-kubeadm-ha-v115.md:9]: I'd add this text to cover the change:
```suggestion

_At the time of publication, Lucas Käldström was writing as SIG Cluster Lifecycle co-chair
and as a subproject owner for `kubeadm`; Fabrizio Pandini was writing as a subproject
owner for `kubeadm`._

```

File: content/en/blog/_posts/2019-06-19-kubernetes-1-15-release-announcement.md [hunk 12]
```diff
@@ -4,8 +4,9 @@ title: "Kubernetes 1.15: Extensibility and Continuous Improvement"
 date: 2019-06-19
 slug: kubernetes-1-15-release-announcement
 evergreen: true
+author: >
+  The 1.15 [Release Team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.15/release_team.md)
```
Comment (sftim) [inline content/en/blog/_posts/2019-06-19-kubernetes-1-15-release-announcement.md:8]:
```suggestion
  [Kubernetes 1.15 Release Team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.15/release_team.md)
```
?

**Comment by sftim** [comment]
/kind cleanup

Much appreciated!

File: content/en/blog/_posts/2019-06-19-kubernetes-1-15-release-announcement.md (hunk 12 above)
Comment (dipesh-rawat) [reply content/en/blog/_posts/2019-06-19-kubernetes-1-15-release-announcement.md]: Thanks for taking time to review. I have applied all the suggestion and review comments as part of the latest commit push.

File: content/en/blog/_posts/2019-07-23-get-started-with-kubernetes-using-python.md [hunk 13]
```diff
@@ -2,10 +2,10 @@
 layout: blog
 title: "Get started with Kubernetes (using Python)"
 date: 2019-07-23
+author: >
+  Jason Haley (Independent Consultant)
```
Comment (sftim) [inline content/en/blog/_posts/2019-07-23-get-started-with-kubernetes-using-python.md:6]:
```suggestion
  Jason Haley (independent consultant)
```

File: content/en/blog/_posts/2019-08-30-announcing-etcd-3.4.md [hunk 14]
```diff
@@ -4,10 +4,11 @@ layout: blog
 title: "Announcing etcd 3.4"
 date: 2019-08-30
 slug: announcing-etcd-3-4
+author: >
+  Gyuho Lee (Amazon Web Services, @[gyuho](https://github.com/gyuho)), 
+  Jingyi Hu (Google, @[jingyih](https://github.com/jingyih))
```
Comment (sftim) [inline content/en/blog/_posts/2019-08-30-announcing-etcd-3.4.md:9]:
```suggestion
  [Gyuho Lee]((https://github.com/gyuho) (Amazon Web Services),
  [Jingyi Hu](https://github.com/jingyih) (Google)
```

File: content/en/blog/_posts/2019-11-05-kubernetes-with-microk8s.md [hunk 15]
```diff
@@ -1,8 +1,10 @@
 ---                                           
 title: 'Running Kubernetes locally on Linux with Microk8s'                                                           
 date: 2019-11-26
+author: >
+  [Ihor Dvoretskyi](https://twitter.com/idvoretskyi) (Developer Advocate, Cloud Native Computing Foundation),
```
Comment (sftim) [inline content/en/blog/_posts/2019-11-05-kubernetes-with-microk8s.md:5]:
```suggestion
  [Ihor Dvoretskyi](https://twitter.com/idvoretskyi) (Cloud Native Computing Foundation),
```

**Review by sftim — COMMENTED** [review]
I've made some more indicative suggestions; please check all the articles in this PR (and any articles from 2019 that this PR doesn't yet cover), and make sure they align with the new way.

**Review by sftim — COMMENTED** [review]
Thanks!

/approve

**Pull request merged**